pkgs/development/python-modules/catppuccin/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python
+, pygments
+}:
+
+buildPythonPackage rec {
+  pname = "catppuccin";
+  version = "1.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-mHNuV3yIuFL2cixDOr+//+/b9iD2fN82cfLzZkegxKc=";
+  };
+
+  propagatedBuildInputs = [ pygments ];
+
+  meta = with lib; {
+    description = "Soothing pastel theme for Python";
+    maintainers = with maintainers; [ fufexan ];
+    license = licenses.mit;
+    homepage = "https://github.com/catppuccin/python";
```

```suggestion
    description = "Soothing pastel theme for Python";
    homepage = "https://github.com/catppuccin/python";
    maintainers = with maintainers; [ fufexan ];
    license = licenses.mit;
```

Missing pythonImportsCheck and tests.